README.md

```diff
@@ -8,6 +8,7 @@ data on a scheduled basis.
 * `aws_region` - AWS region to run in, default: `us-east-1`
 * `aws_access_key` - AWS access key
 * `aws_secret_key` - corresponding AWS secret
+* `app_env` - application environment, default: `prod`
```

Comment:
This works, but you could also do everything without an explicit `app_env` variable. See below.